src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -40,7 +40,16 @@ contract BountyQuestion is Ownable, OnlyApi {
         return ret;
     }
 
+    function getAuthorOfQuestion(uint256 questionId) public view returns (address) {
+        return questions[questionId].author;
```

Comment:
if a question is created as a "challenge" by an admin the author is irrelevant - I don't think that matters and this code is good!  just a use-case to keep in mind